### PR TITLE
fix: rely on configurationValues if it has the key value

### LINKED ISSUE #9767: Podman Machine create sliders are not visible when having both HyperV and WSL dropdown present
### Bug description

Have both WSL and hyperV features enabled and podman desktop started with admin rights. Open Create new Machine and then try to noticed the sliders that should be available for hyperv option once it is selected in the dropdown. 

Image

### Operating system

Windows 10

### Installation Method

Installer from website/GitHub releases

### Version

1.14.0

### Steps to reproduce

_No response_

### Relevant log output

_No response_

### Additional context

When I have only a hyperv feature enabled, I can see the sliders.

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This patch fixes an issue when retrieving the values from the "create new machine" form that are used when auditing and then update the UI in certain scenarios (like hyperv/wsl)

### Screenshot / video of UI

hyperv_create_3

### What issues does this PR fix or reference?

it fixes #9767 

### How to test this PR?

you can follow the issue explained at https://github.com/podman-desktop/podman-desktop/issues/9767
1.  run desktop as admin
2. go to create new page
3. select wsl or hyperv and play with the sliders/create new machine

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Comment by lstocchi** [comment]
cc @odockal

**Review by jeffmaury — APPROVED** [review]
LGTM. Is there a way to have a unit test ?

**Comment by lstocchi** [comment]
@jeffmaury busy with other things atm. If I'm not able to add them today I'll look at them tomorrow morning

**Review by axel7083 — APPROVED** [review]

File: packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.spec.ts [hunk 1]
```diff
@@ -391,6 +392,59 @@ describe.each([
   });
 });
 
+test(`Check itemsAudit receive updated values`, async () => {
+  const callback = mockCallback(async () => {});
+  const auditSpy = vi.spyOn(window as any, 'auditConnectionParameters').mockImplementation(() => ({ records: [] }));
```
Comment (jeffmaury) [inline packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.spec.ts]:
```suggestion
  const auditSpy = vi.spyOn(window as any, 'auditConnectionParameters').mockResolvedValue({ records: [] });
```

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by gastoner** [comment]
@mergifyio backport 1.14.x

**Pull request merged**